gce/prepull.yaml

```diff
@@ -0,0 +1,136 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: prepull
+spec:
+  selector:
+    matchLabels:
+      prepull-test-images: e2e
+  template:
+    metadata:
+      labels:
+        prepull-test-images: e2e
+    spec:
+      nodeSelector:
+        kubernetes.io/os: windows
+      initContainers:
```

Comment:
oh that's clever :)

Reply:
cc @adelina-t

Reply:
Nice, I'll test this to see how it works for us in aks-engine